estruturacao.py

```diff
@@ -0,0 +1,40 @@
+import sys
+
+
+def lerArquivo(arquivo):
+	arq = open(arquivo, 'r')
+	test = arq.readlines()
+	test = [ x.replace("\n", "").split(" ") for x in test]
```

Comment:
O que acha de utilizar somente o `split()`, chamando ele sem passar parâmetros ele já realiza o split nos " " e "\n"
algo como:
```suggestion
	test = [x.split() for x in test]
```

Reply:
Considerando o comentário lá em baixo, da função que já retorna o array com floats e int. podemos chamar essa função aqui! Já passando o split como parâmetro.

```suggestion
	test = [convert_to_number(x.split()) for x in test]
```